includes/discovery/sensors/current/junos.inc.php

```diff
@@ -16,7 +16,7 @@
     $multiplier = 1;
     $divisor    = 1000000;
     foreach ($junos_oids as $index => $entry) {
-        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent'])) {
+        if (is_numeric($entry['jnxDomCurrentTxLaserBiasCurrent']) && $entry['jnxDomCurrentTxLaserBiasCurrent'] && $entry['jnxDomCurrentTxLaserBiasCurrentLowAlarmThreshold'] != 0) {
```

Comment:
No we are skipping the sensor discovery if $entry['jnxDomCurrentTxLaserBiasCurrent'] != 0 AND $entry['jnxDomCurrentTxLaserBiasCurrentLowAlarmThreshold'] != 0.

The idea by doing this is to catch invalid entry's like the ones pasted in the PR.

Forgive me if I'm not getting your point.